### DISCUSSION TITLE
Group updates per artifact group

### DISCUSSION THREAD

**Discussion by adriil** [renovatebot/renovate#19196] [Request Help] [answered]
Hi team,

Is there a generic way to tell Renovate Bot to group all updates per artifact group, **without hardcoding any group** ?

For instance, with the following libraries...
```
group1:nameA:version1
group1:nameB:version2
group2:nameC:version3
group2:nameD:version4
```
... I'd like to have two PRs for each `group1` and `group2`, but without hardcoding all the possible groups like here
```json
"packageRules": [
  {
    "matchPackagePatterns": ["group1"],
    "groupName": "group1 updates",
  },
  {
    "matchPackagePatterns": ["group2"],
    "groupName": "group2 updates",
  }
]
```

Thanks,
Adrien

**Comment by rarkins** [comment, marked as answer]
I don't think we support this currently
